scripts/lazygit_auto_commit.sh

```
@@ -0,0 +1,246 @@
+        log_message "現在のペイン情報:"
+        tmux display-message -p "#{session_name}:#{window_index}.#{pane_index}" 2>/dev/null | while read pane; do
+            log_message "  現在のペイン: $pane"
+        done
+    fi
+    
+    log_message "tmux環境でコミットキー送信を実行"
+    if tmux send-keys c 2>/dev/null; then
+        success=true
+        log_message "tmux環境でコミットキー送信成功"
+    else
+        local tmux_error=$?
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)